.travis.yml

```diff
@@ -11,6 +11,8 @@ matrix:
           env: KERAS_BACKEND=theano
         - python: 2.7
           env: KERAS_BACKEND=tensorflow
+        - python: 2.7
+          env: KERAS_BACKEND=theano INTEGRATION_TESTS=true
```

Comment:
Wouldn't it be preferable to run the integration tests with TensorFlow, since compilation times are considerably reduced (except for RNNs)?